src/mixins/resource-lookup-mixin.ts

```diff
@@ -3,22 +3,23 @@ import { Component, Vue } from 'vue-property-decorator'
 import { State } from 'vuex-class'
 
 // Interfaces
-import { ResourceExampleIF } from '@/interfaces'
+import { ExternalResourceIF } from '@/interfaces/resource-interfaces/ExternalResourceIF'
```

Comment:
Should be able to just import from `@/interfaces` if the index is exporting * from the specific interface folder.